docs/docusaurus/docs/cloud/about_gx.md

```diff
@@ -1,26 +1,20 @@
 ---
-sidebar_label: 'About GX Cloud'
-title: 'About GX Cloud'
-description: Learn more about the features and benefits of GX Cloud.
+sidebar_label: 'GX Cloud deployment patterns and architecture'
+title: 'GX Cloud deployment patterns and architecture'
+description: Learn more about GX Cloud deployment patterns and architecture.
 toc_min_heading_level: 2
 toc_max_heading_level: 2
 ---
 
-GX Cloud builds on the innovative features of GX open source software (OSS). Its easy-to-use web interface lets you quickly validate your data without creating and running complex Python code. With GX Cloud you can work collaboratively with your teammates to define and test reusable data queries that alert you to changes in your data.
+GX Cloud simplifies data quality management and monitoring. Its easy-to-use web interface lets you quickly validate your data without creating and running complex Python code. With GX Cloud, you can work collaboratively with your teammates to define and test reusable data queries that alert you to changes in your data.
 
-While it is possible to design and deploy a functioning data quality ecosystem with GX OSS, GX Cloud simplifies infrastructure and deployment significantly. With GX Cloud, you'll save time, money, and effort implementing your data quality management and monitoring solution.
-
-Here are just a few of the benefits you'll get with GX Cloud:
+GX Cloud includes the following features and functionality:
```

Comment:
This list is thin. I'm sure we can flesh out this list of Cloud benefits. Will you please reach out to Thaise to get ideas?

This doesn't need to block this PR.